docs/README.md

```diff
@@ -59,6 +58,16 @@ The credentials can be supplied by providing the `modules.core.aws` setting in t
 }
 ```
 
+AWS SDK can be also configured in code by using `altis.aws_sdk.params` filter which is called right before the global instance of the AWS SDK is created.
+
+```
+add_filter( 'altis.aws_sdk.params', 'aws_sdk_params' );
+
+function aws_sdk_params( $params ) {
+	return $params['region'] = 'new_region';
+}
+```
```

Comment:
In Filters section it's mentioned that these should be called early. Please let me know what's the correct priority to call this on and I'll update the docs. Thanks!

Reply:
That's more for anyone using the filter rather than a priority to set here.

Reply:
Good shout, it might be good to note in the paragraph above this example that this filter can only be used in the bootstrap function of an Altis module as it's called very early in the loading process.